.github/workflows/main.yml

```diff
@@ -45,10 +48,9 @@ jobs:
       - name: build cache statistics
         run: sccache --show-stats
       - name: Cleanup
-        shell: bash
         if: always()
         run: |
-          # Do not cache temporary envs with 'cache-env: true'
+          # Do not cache temporary envs with 'cache-environment: true'
```

Comment:
I think we don't need this step anymore. In setup-micromamba we changed the point in time when the environment is cached to right after environment creation, not after the rest of the CI.